horace_core/algorithms/gen_sqw_cylinder.m

```diff
@@ -1,5 +1,10 @@
-function gen_sqw_cylinder_test_helper (spe_file, par_file, sqw_file, efix, varargin)
+function gen_sqw_cylinder(spe_file, par_file, sqw_file, efix, varargin)
 % Read one or more spe files and a detector parameter file, and create an output sqw file.
+% 
+% WARNING: The sqw objects and files, produced by this routine are not
```

Comment:
this warning isn't clear -- what does "not fully functional" mean? Is that what's covered by the `*** IMPORTANCE NOTES ***` below.

Should the 
```
	% *** TEST ROUTINE
%       This was created rapidly as a fix-up during an experiment. A polished version is	%       This was created rapidly as a fix-up during an experiment. A polished version is
%       marked for addition at a later date.	%       marked for addition at a later date.
```
comment be deleted? This isn't a test routine as more as it's moved into the main codebase and won't be used for testing; it's a partial implementation of the functionality with limitations noted in the header and reinforced by the `WARNING`

Reply:
(same comment for `gen_sqw_powder`)

Reply:
I have no idea and no desire to find out -- our plan is to fully redesign this code. The advice is to look at the tests and see what is working.

Reply:
can be a bit rephrased -- yes.